docs/src/api/class-browsercontext.md

```diff
@@ -478,6 +542,26 @@ page.set_content("""
 """)
 ```
 
+```csharp
+var result = new TaskCompletionSource<string>();
+var page = await Context.NewPageAsync();
+await Context.ExposeBindingAsync("clicked", async (BindingSource _, IJSHandle t) =>
```

Comment:
@dgozman We already have a `Func<BindingSource, T, TResult>` overload :)
So you can run `async` code, but that `Task` won't be awaited inside our handler.

Reply:
Nice!